libs/textutil/textutil.go

```diff
@@ -0,0 +1,20 @@
+package textutil
+
+import (
+	"strings"
+	"unicode"
+)
+
+// We leave the full range of unicode letters in tact, but remove all "special" characters,
+// including spaces and dots, which are not supported in e.g. experiment names or YAML keys.
+func NormaliseString(name string) string {
```

Comment:
This is British spelling. We use American in other places (e.g. `initialize`). -> Normalize to fix.